doc/source/user/basics.copies.rst

```diff
@@ -0,0 +1,135 @@
+.. _basics.copies-and-views:
+
+****************
+Copies and views
+****************
+
+The NumPy array is a contiguous block of memory consisting of two parts- the
+data buffer with the actual data elements and the metadata which contains
+information about the data buffer. The metadata includes data type, strides
+and other important information that helps manipulate the :class:`.ndarray`
+easily. See the :ref:`organization-numpy-arrays` section for a detailed look.
+
+View or shallow copy
+====================
+
+It is possible to perceive the array differently by changing the
+metadata. These new arrays are called views or shallow copies. The data buffer
+remains the same so any changes made to a view reflects in the original
+copy. A view can be forced through the :meth:`.ndarray.view` method. 
```

Comment:
I'm not sure I understand the first sentence. I think you mean that changing the metadata can change how you access the data buffer without changing its internal layout in memory, is that it? Maybe reformulate...